apis/infix-en_GB/atrium-api-infix-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/DateSubjectChangerSamples.kt

```diff
@@ -43,4 +44,37 @@ class DateSubjectChangerSamples {
             }
         }
     }
+
+    @Test
+    fun asLocalDateTimeFeature() {
+        val date = formatter.parse("1995-07-17")
+
+        expect(date) asLocalDateTime o toEqual(LocalDateTime.parse("1995-07-17T00:00:00")) // Subject is now of type LocalDateTime
```

Comment:
```suggestion
        expect(date) asLocalDateTime o toEqual LocalDateTime.parse("1995-07-17T00:00:00") // Subject is now of type LocalDateTime
```
Would you mind to change asLocalDateFeature accordingly as well?